### DISCUSSION TITLE
Problems comparing Word files

### DISCUSSION THREAD

**Discussion by Walter-VR** [WinMerge/winmerge#1749] [General]
I use WinMerge frequently to synchronize backup copies of my personal files.
Normally, I use the option to compare date and size, but once in a while I prefer to use the full compare.

This works perfectly to compare directories from my PC to a copy on an external hard disk.

When I use it to compare directories on an USB stick however, there are problems with a lot of Word files (doc or docx). The compare stops with a popup error message to tell me that "a serious error occurred since the last time the file was opened". When I click on "yes" the file opens anyway and the process continues until the next error.

This takes a lot of time and requires my continuous presence to give a few hundred clicks.
Is there a way to avoid this? I am thinking on a "yes to all" option to accept the error and to continue anyway.[
![wm2](https://user-images.githubusercontent.com/127603193/224485047-2e4f2e81-2cf6-45df-94f5-e930635b6365.jpg)
![wm1](https://user-images.githubusercontent.com/127603193/224485048-76300e6b-733f-4b97-afa2-a5576e1429bc.jpg)
](url)

**Comment by sdottaka** [comment]
It appears that the CompareMSWordFiles plugin is automatically applied.
If you simply want to make sure that the Word files match, you do not need to apply the plugin.

If you check Plugins→Manual Unpacking menu item, the plugin will not be applied automatically and you will not see this message box.

Alternatively, if you change the folder compare method from Full Contents to Binary Contents, the plugin will no longer be applied.

![image](https://user-images.githubusercontent.com/98126/224515134-af2b76a1-b2fa-4343-8131-6b6761ba797a.png)

**Reply by Walter-VR** [reply]
Thanks very much for the fast reply!
